resources/fixtures/trade.json

```diff
@@ -24,7 +24,7 @@
   "counter_asset_issuer": "GCKA6K5PCQ6PNF5RQBF7PQDJWRHO6UOGFMRLK3DYHDOI244V47XKQ4GP",
   "base_is_seller": true,
   "price": {
-    "N": 10,
-    "D": 61
+    "n": 10,
```

Comment:
I think we had it upper case because the documentation here:
https://www.stellar.org/developers/horizon/reference/endpoints/trades.html

The example json has uppercases 🤷‍♂️

Reply:
Hmmm that is weird.

See here: https://www.stellar.org/laboratory/#explorer?resource=trades&endpoint=all&values=eyJiYXNlX2Fzc2V0Ijp7InR5cGUiOiIiLCJjb2RlIjoiIiwiaXNzdWVyIjoiIn0sImNvdW50ZXJfYXNzZXQiOnsidHlwZSI6Im5hdGl2ZSIsImNvZGUiOiIiLCJpc3N1ZXIiOiIifX0%3D&network=test

Reply:
Yea, wouldn't be the first time we've seen the response structure be different in the API versus in the documentation.

Reply:
It'd be awesome if we note these :)

Reply:
Absolutely we should note these. Where is the best place to do that? Is the documentation in a public repo?

Reply:
I think it might be, I'll give it a look tomorrow at work